app/models/ride/ride.ts

```diff
@@ -52,11 +53,17 @@ export const RideModel = types
         frequentPushesEnabled: types.boolean,
       }),
     ),
-  })
-  .views((self) => ({
     /**
-     *
+     * Notifee settings for android
      */
+    notifeeSettings: types.maybe(
+      types.model({
+        notifications: types.number,
+        alarms: types.number,
+      }),
+    ),
+  })
```

Comment:
@planecore I don't think this should be in this model. We have a `settings` model which is more suitable.

Reply:
It's in the same place with the iOS equivalent `activityAuthorizationInfo`